src/datacollector/app.config

```diff
@@ -0,0 +1,16 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <startup useLegacyV2RuntimeActivationPolicy="true">
+    <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.0" />
+  </startup>
+  <runtime>
+    <legacyUnhandledExceptionPolicy enabled="1" />
+    <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1">
```

Comment:
`This config is auto generated for netcore, right?`
@codito : Yes.

`Is this config file included in the testplatform.vsix? and our nuget packages?`
@AbhitejJohn : Yes.